mobile_verifier/src/heartbeats/mod.rs

```diff
@@ -225,10 +225,10 @@ impl Heartbeat {
         }
     }
 
-    pub fn asserted_distance(&self, asserted_location: u64) -> anyhow::Result<i64> {
```

Comment:
wondering why there was no change to the caller to `asserted_distance` but seems there isnt one

Reply:
yeah, function wasn't being used, so I killed it